Cargo.toml

```diff
@@ -10,6 +10,6 @@ documentation = "https://docs.rs/plane-split"
 
 [dependencies]
 binary-space-partition = "0.1.2"
-euclid = "0.13"
+euclid = "~0.14.1"
```

Comment:
my understanding is that `~0.14.1` means "between 0.14.1 and 0.15 not included" whereas `0.14.1` means "specifically version 0.14.1".